test/test-network/systemd-networkd-tests.py

```diff
@@ -3481,6 +3501,54 @@ def test_ipv6_proxy_ndp(self):
         for i in range(1, 5):
             self.assertRegex(output, f'2607:5300:203:5215:{i}::1 *proxy')
 
+    def test_ipv6_neigh_retrans_time(self):
+        link='test25'
+        copy_network_unit('25-dummy.netdev', '25-dummy.network')
```

Comment:
I do not think `25-dummy.network` is necessary. Why not to copy `25-ipv6-neigh-retrans-time-3s.network` at first?

Reply:
Probably not necessary, but it forms a nice pattern.